packages/web/util.test.js

```diff
@@ -0,0 +1,113 @@
+import { findJSONLocationOfMonacoText, findMonacoLocationOfJSONText } from "./util"
+
+const testCases = [
```

Comment:
it would be better to rename this since it is only about monaco pointer. I would suggest `monacoPointerTestCases`

I cannot do a github based suggestion since it is used in other parts of the code

Reply:
fixed with commit [ce9e639](https://github.com/thingweb/thingweb-playground/pull/360/commits/ce9e639c31c39b1ac3f0fbb7db88794fcaaa2497)